youtube_dl/extractor/vier.py

```diff
@@ -105,13 +126,13 @@ def _real_extract(self, url):
         entries = []
         for current_page_id in itertools.count(start_page):
             current_page = self._download_webpage(
-                'http://www.vier.be/%s/videos?page=%d' % (program, current_page_id),
+                'http://www.%s.be/%s/videos?page=%d' % (site, program, current_page_id),
                 program,
                 'Downloading page %d' % (current_page_id + 1))
             page_entries = [
-                self.url_result('http://www.vier.be' + video_url, 'Vier')
+                self.url_result('http://www.'+site+'.be' + video_url, site.title())
```

Comment:
Oops, I didn't get that, and the tests did not catch that. I guess Idid a too literal search and replace operation.